pom.xml

```diff
@@ -626,12 +626,32 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
-            <distributionManagement>
-                <site>
-                    <id>reports</id>
-                    <url>${reports.repository.url}/${version}</url>
-                </site>
-            </distributionManagement>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-source-plugin</artifactId>
```

Comment:
I'm probably misunderstanding the maven-source-plugin. What is the purpose of adding it here?